common/test/acceptance/tests/lms/test_lms_instructor_dashboard.py

```diff
@@ -425,7 +426,7 @@ def test_can_add_remove_allowance(self):
         # Then, the added record should be visible
         self.assertTrue(allowance_section.is_allowance_record_visible)
 
-    @flaky  # See EDUCATOR-551
+    @skip("EDU-551, EDU-949")
```

Comment:
Is there a length limit that keeps you from writing this out?  I think it is cleaner to be able to copy/paste and have an actual ticket id.